srv/isc-agent/plugins/tcp/http/main.py

```diff
@@ -1,7 +1,12 @@
+import logging
+from http import HTTPStatus
 from twisted.web import server, resource
 from twisted.internet import reactor, endpoints
 from twisted.web.http import Request
 
+PRODSTRING = 'Apache/3.2.3'
```

Comment:
I'm not sure this is even needed but let's keep it for now but move it into the settings. Let's make it so this value is configurable through http plugin in the settings file.

You can choose do it on this story or ISC-15